### PR TITLE
Add t2v command to cli v1

### PR DESCRIPTION
**Pull request by pyamin1878**

### CODE REVIEW

File: aictl/cli/main.py [hunk 1]
```diff
@@ -112,6 +112,36 @@ def ip2p(args):
...
+    else:
+        pipe.enable_vae_slicing()
+        pipe.enable_model_cpu_offload()
+        pipe.enable_xformers_memory_efficient_attention()
+    pipe = pipe.to(device)
+     
+    print(args.prompt)
+    video_frames = pipe(args.prompt, num_frames=args.frames).frames
+    video_path = export_to_video(video_frames)
+    print(video_path)
```
Comment (trizko) [inline aictl/cli/main.py:144]: Is there anyway we can add an `--output-path` argument here to allow the user to specify where they want the file and what name they want to use?

File: aictl/cli/main.py [hunk 2]
```diff
@@ -112,6 +112,36 @@ def ip2p(args):
...
+
+     # check if on mac and mps is available, fallback to cuda then cpu
+    is_mac = False
+    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+    if torch.backends.mps.is_available():
+        is_mac = True
+        device = torch.device("mps")
+        print("MPS device detected. Using MPS.")
+
+    pipe = DiffusionPipeline.from_pretrained("damo-vilab/text-to-video-ms-1.7b", torch_dtype=torch.float16, variant="fp16")
```
Comment (trizko) [inline aictl/cli/main.py]: float16 will not work on Mac so what I did for the other commands was to add this line to switch to float32 when a Mac is detected:
```python
model_type = torch.float32 if is_mac else torch.float16
```
Then you would change this line to:
```python
pipe = DiffusionPipeline.from_pretrained("damo-vilab/text-to-video-ms-1.7b", torch_dtype=model_type)
```

Comment (trizko) [inline aictl/cli/main.py]: Instead of hardcoding the model name here, we can use the `args.model` argument supplied by the user.

**Review by trizko — COMMENTED** [review]
I left a couple comments on the PR. This looks great so far.

**Comment by pyamin1878** [comment]
Thanks for the review, I updated the PR to address your comment ^_^

**Comment by trizko** [comment]
This looks great! Thank you so much for this contribution.

**Pull request merged**